src/xibo-interactive-control.js

```diff
@@ -438,7 +439,7 @@ window.xiboIC = (function() {
         // Backup value as data
         $viewPortEl.data('viewportValueBackup', originalValue);
         $viewPortEl.attr('content',
-          originalValue + ' maximum-scale=1.0, user-scalable=no');
+          originalValue + ', maximum-scale=1.0, user-scalable=no');
```

Comment:
I think this has to be optional if there is a value already there?

Reply:
Oh I see, so the `originalValue` can come as empty, I'll fix that, thanks! :)